src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/tests/__snapshots__/Checkbox.test.js.snap

```diff
@@ -1,50 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
-
-exports[`The component should render in dark skin 1`] = `
```

Comment:
Looks like not all snapshots where regenerated. The CI seems to fail because of that.